packages/wallets/src/components/WalletsActionScreen/WalletsActionScreen.tsx

```diff
@@ -50,9 +50,13 @@ const WalletsActionScreen: React.FC<PropsWithChildren<TProps>> = ({
                         {title}
                     </WalletText>
                 )}
-                <WalletText align='center' size='md'>
-                    {description}
-                </WalletText>
+                {isValidElement(description) ? (
+                    description
+                ) : (
+                    <WalletText align='center' size='md'>
+                        {description}
+                    </WalletText>
+                )}
```

Comment:
Added condition for description which wraps the content in a `WalletText` component when description is a string and not wrap it when it is a function. This would remove the rendering of an additional `WalletText` component when it is unneeded